### PR TITLE
feat: [STO-6477]: Add missing STO_ FFs

### PR DESCRIPTION
**Pull request by GabeAboy**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/10-common/featureFlags.ts [hunk 1]
```diff
@@ -108,9 +108,21 @@ export enum FeatureFlag {
   ENABLE_K8_BUILDS = 'ENABLE_K8_BUILDS',
   CD_TRIGGER_CATALOG_API_ENABLED = 'CD_TRIGGER_CATALOG_API_ENABLED',
   FF_FLAG_SYNC_THROUGH_GITEX_ENABLED = 'FF_FLAG_SYNC_THROUGH_GITEX_ENABLED',
+  STO_AI_ENHANCED_REMEDIATIONS = 'STO_AI_ENHANCED_REMEDIATIONS',
   STO_ALL_ISSUES_PAGE = 'STO_ALL_ISSUES_PAGE',
-  STO_BASELINE_REGEX = 'STO_BASELINE_REGEX',
```
Comment (chris-harness) [inline src/modules/10-common/featureFlags.ts:112]: Don't remove `STO_BASELINE_REGEX`

**Comment by GabeAboy** [comment]
trigger commonchecks

File: src/modules/10-common/featureFlags.ts (hunk 1 above)
Comment (GabeAboy) [reply src/modules/10-common/featureFlags.ts:112]: Resolving this

**Review by chris-harness — APPROVED** [review]

**Review by jsentongo — APPROVED** [review]

**Comment by GabeAboy** [comment]
retrigger cypress-pipeline

**Pull request merged**